### ISSUE TITLE
Docking frames outside of main windows looks funky

### ISSUE THREAD

**Issue by Tdue21** [rtakehara/5e-Framework#55]
**Describe the bug**
With a multiple monitor setup, it is possible to place frames etc. outside of the main window for maximizing map viewport. 
The result is less than aesthetical pleasing to look at. 

I realize it is probably an artifact of the docking system, but hopefully something that can be tweaked by someone than actually understands Java (i.e. not me ;) ).

When two or more frames are docked together outside of the main windows, they are first framed by a Windows XP looking inner window, and then the native window (Windows 10, in my case). 

![image](https://user-images.githubusercontent.com/2953375/163191298-0bfb598e-2eb3-45aa-b809-60671ac1562b.png)

**To Reproduce**
1. Drag two frames outside of the main window
2. Dock the two frames together.
3. Observe a funky looking window.

**Expected behavior**
Basically what happens now, but without the extra Windows XP looking inner window. 

**Desktop (please complete the following information):**
 - OS: Windows 10 Home
 - Maptool Version 1.11.5

**Comment by Tdue21** [comment]
Oh.. sorry, wrong project... Moving this to MapTool .. ;)
